sdk/vppagent/dataplane/span.go

```diff
@@ -0,0 +1,47 @@
+package dataplane
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/opentracing/opentracing-go"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/crossconnect"
+	"github.com/networkservicemesh/networkservicemesh/dataplane/pkg/apis/dataplane"
+	"github.com/networkservicemesh/networkservicemesh/sdk/common"
+)
+
+type span struct {
```

Comment:
Please do this similar to how its done in the SDK:

https://github.com/networkservicemesh/networkservicemesh/blob/master/sdk/endpoint/composite.go#L61

Reply:
thanks, fixed